core/types/calls.cc

```diff
@@ -1099,12 +1099,45 @@ class T_Generic_squareBrackets : public IntrinsicMethod {
         auto it = args.args.begin();
         int i = -1;
         targs.reserve(attachedClass.data(ctx)->typeMembers().size());
+        bool validBounds = true;
         for (auto mem : attachedClass.data(ctx)->typeMembers()) {
             ++i;
-            if (mem.data(ctx)->isFixed()) {
-                targs.emplace_back(mem.data(ctx)->resultType);
+
+            auto memData = mem.data(ctx);
+
+            // TODO: why is mem.data(ctx)->resultType sometimes nullptr?
```

Comment:
I'm also curious :-)

Reply:
I think I'm going to need to get to the bottom of this to be able to remove uses of `isFixed` as we had talked about earlier today.

Reply:
This was happening because I had missed a place that type members were processed. As a result, they were getting non-LambdaParam resultType values in their Symbols.